docs/automatic-installation.md

```diff
@@ -0,0 +1,255 @@
+# Automatic Installation
+
+Harvester can be installed in a mass manner. This document provides an example to do the automatic installation with PXE boot.
+
+We recommend using [iPXE](https://ipxe.org/) to perform the network boot. It has more features than the traditional PXE Boot program and is likely available in modern NIC cards. If NIC cards don't come with iPXE firmware, iPXE firmware images can be loaded from the TFTP server first.
+
+## Preparing HTTP servers
+
+An HTTP server is required to serve boot files. Please ensure these servers are set up correctly before continuing.
+
+Let's assume an nginx HTTP server's IP is `10.100.0.10`, and it serves `/usr/share/nginx/html/` folder at path `http://10.100.0.10/`.
+
+## Preparing boot files
+
+- Download the [Harvester ISO](https://github.com/rancher/harvester/tags).
+
+- Serve the ISO file.
+  
+  Copy or move the ISO file to an appropriate location so it can be downloaded via the HTTP server. e.g.,
+
+  ```
+  sudo mkdir -p /usr/share/nginx/html/harvester/
+  sudo cp /path/to/harvester-amd64.iso /usr/share/nginx/html/harvester/
+  ```
+
+- Extract kernel and initrd file from the ISO file.
```

Comment:
We can remove this step given that we publish these files now

Reply:
Fixed. Thanks.